### ISSUE TITLE
Focus not set on search box if last search in Viewer was for Hex

### ISSUE THREAD

**Issue by rohitab** [FarGroup/FarManager#605]
### Far Manager version

3.0.6090.0 x64

### OS version

10.0.22623.1095

### Steps to reproduce

1. Open any file in the internal Viewer using `F3`
2. Press `F7` to open _Search_
3. Press `Alt+H` to switch to _Hex_ mode
4. Type in some hex data in the _Search for_ box
5. Press `Enter` to start the search
6. If you get a _Could not find bytes message_, press `Enter`
7. Press `F7` to reopen _Search_

### Expected behavior

Focus should be on the _Search for_ edit control.

### Actual behavior

Focus is on the _Text_ radio button.

You need to press `Tab` twice, or use `Alt+S` to get to the _Search for_ box. Note that if the previous search was in _Text_ mode, and you reopen search, the focus is correctly set on the _Search for_ box.

Also, if you use the `Alt+T` and `Alt+H` hotkeys to switch between _Text_ and _Hex_ mode, the focus remains on the radio buttons, unlike the _Find file_ dialog, where the focus is moved to the _Containing_ search box.

**Comment by MKadaner** [comment]
Of course, it does not work; it has never worked this way. 🤣

Thank you for reporting. Fixing.

**Comment by MKadaner** [comment]
Should be fixed in 6091.

**Comment by rohitab** [comment]
I just tested version `3.0.6092.0 x64`. The issue is now fixed. Thank you.

I just noticed another minor difference between the Find file dialog and the _Search_ dialog in the Viewer, when using the mouse to switch between _Text_ and _Hex_ modes. In the _Find file_ dialog, the focus remains on the radio buttons. In the _Search_ dialog of the Viewer, the focus is moved to the _Search for_ box. However, this only happens after you use the `Alt+T` and `Alt+H` hotkeys at least once. Before using the hotkeys, the focus remains on the radio buttons.

It's almost a non issue, since most people will either use the mouse or the hotkeys, but rarely both. That's why I'm reporting it here instead of creating a new ticket.
